src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/EventHandlers/ApprenticeshipConfirmationCommencedEventHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Threading.Tasks;
+using MediatR;
+using NServiceBus;
+using SFA.DAS.CommitmentsV2.Application.Commands.ApprenticeshipConfirmationCommenced;
+
+namespace SFA.DAS.CommitmentsV2.MessageHandlers.EventHandlers
+{
+    public class ApprenticeshipConfirmationCommencedEventHandler : IHandleMessages<ApprenticeshipConfirmationCommencedEvent>
+    {
+        private readonly IMediator _mediator;
+
+        public ApprenticeshipConfirmationCommencedEventHandler(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        public Task Handle(ApprenticeshipConfirmationCommencedEvent message, IMessageHandlerContext context)
+        {
+            return _mediator.Send(new ApprenticeshipConfirmationCommencedCommand(message.CommitmentsApprenticeshipId,
+                message.CommitmentsApprovedOn, message.ApprenticeshipConfirmationOverdueOn));
+        }
+    }
+
+    public class ApprenticeshipConfirmationCommencedEvent
+    {
+        public Guid ApprenticeId {get; set;}
```

Comment:
is this required?

Reply:
No, it's not being used at the moment. But because this belongs to another domain. The object won't be here